jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Checkbox.java

```diff
@@ -0,0 +1,83 @@
+package com.epam.jdi.light.material.elements.inputs;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.material.asserts.inputs.CheckboxAssert;
+import org.openqa.selenium.By;
+
+public class Checkbox extends UIBaseElement<CheckboxAssert> {
+
+    @JDIAction("Click on '{name}'")
+    public void clickOnCheckbox() {
+        core().click();
+    }
+
+    @JDIAction("Is '{name}' selected")
+    public boolean isSelectedCheckbox() {
+        return core().hasClass("Mui-checked");
+    }
+
+    @JDIAction("Is '{name}' not selected")
+    public boolean isNotSelectedCheckbox() {
+        return !isSelectedCheckbox();
+    }
+
+    @JDIAction("Is '{name}' selected")
+    public boolean isEnabledCheckbox() {
+        return core().isEnabled();
+    }
+
+    @JDIAction("Check '{name}'")
+    public boolean check() {
+        if (isSelectedCheckbox()) {
+            return true;
+        } else
+            return false;
+    }
+
+    @JDIAction("Do not check '{name}'")
+    public boolean unCheck() {
+        if(isNotSelectedCheckbox()) {
+            return true;
+        } else
+            return false;
+    }
+
+    @JDIAction("Is '{name}' disabled")
+    public boolean isDisabledCheckbox() {
+        return core().hasClass("Mui-disabled");
+    }
+
+    @JDIAction("Is'{name}' displayed")
+    public boolean isFormControlDisplayed() {
```

Comment:
from control is another component
it wraps input, label and etc.
common checkbox does not contain such structure
should be removed or moved to another class for such controls